README.md

```diff
@@ -137,6 +137,7 @@ kafka.consumers.cache.size=50
 #Kafka sensor properties. Not all are required. Adjust according to your use case.
 kafkaSource.group.id.prefix=hyper_drive_${appUniqueId}
 kafkaSource.poll.duration=500
+kafkaSource.always.seek.to.end=false
```

Comment:
I like `kafkaSource.always.catchup` better. I see why you named it `always.seek.to.end`, but it seems not straight-forward to understand

Reply:
Done